captum/optim/_param/image/images.py

```diff
@@ -136,9 +137,9 @@ class ToRGB(nn.Module):
     def klt_transform():
```

Comment:
Isn't `ToRBG` also a transformation and we could move it to `/image/transform.py` ?

Reply:
Yeah I guess it is! I've moved it to `transform.py`.